mapclientplugins/geometryfitter/view/geometryfitterwidget.py

```diff
@@ -367,12 +367,23 @@ def _makeConnectionsDisplay(self):
         self._ui.displaySurfacesExterior_checkBox.clicked.connect(self._displaySurfacesExteriorClicked)
         self._ui.displaySurfacesTranslucent_checkBox.clicked.connect(self._displaySurfacesTranslucentClicked)
         self._ui.displaySurfacesWireframe_checkBox.clicked.connect(self._displaySurfacesWireframeClicked)
+        self._setupGroupDisplayWidgets()
+    
+    def _setupGroupDisplayWidgets(self):
+        """
+        Set up group display widgets and display values from fitter object.
+        """
+        self._ui.groupDisplay_fieldChooser.setRegion(self._fitter.getRegion())
+        self._ui.groupDisplay_fieldChooser.setNullObjectName("-")
```

Comment:
I think this could be `self._ui.groupDisplay_fieldChooser.setNullObjectName("All")`

Reply:
There could well be a group called "All" (as it's a good name for a group you'd put all elements in).
I'm not sure that matters -- this text simply appears first and represents None. Would be good to confirm the logic works.
Perhaps it should read "All groups" which is really unlikely to be used.
Then the UI would read "Group: [All groups]".

Reply:
At one point I spent several hours trying to display that special entry in italics to make it look distinct from the actual group names, but I couldn't get it to work so went back to "-".

Reply:
I can try to make the null object show in italics by editing the fieldchooserwidget in zintwidgets, it will affect all the other fieldchoosers.

Reply:
I think something like `- All -` would work as a first step.